op-deployer/pkg/deployer/verify/etherscan.go

```diff
@@ -46,6 +46,8 @@ func getAPIEndpoint(l1ChainID uint64) (string, error) {
 		return "https://api-optimistic.etherscan.io/api", nil // optimism-mainnet
 	case 8453:
 		return "https://api.basescan.org/api", nil // base-mainnet
+	case 57073:
+		return "https://explorer.inkonchain.com/api", nil // ink-mainnet (blockscout)
```

Comment:
The command ran successfully even though this a blockscout explorer (which is not officially supported according to the docs). 

I think the two APIs are equivalent, so it should "just work".

Reply:
Although the command successfully ran, behind the scenes it looks like there was actually a problem.
I added some more debug logging, and saw `Poll verification status {1 OK Fail - Unable to verify}`

I wonder if blockscout returns "1" for failure?